From d2e31151159ca6317ae5f66191b196a264099903 Mon Sep 17 00:00:00 2001
From: Jan Engelhardt <jengelh@inai.de>
Date: Sat, 13 Sep 2014 15:06:39 +0200
Subject: [PATCH 39/76] prefs: disable "Use a prediction service to help
 complete searches..."

This autocomplete service would send search queries as soon as you
started typing, even if not yet done typing. Some regard this as an
information leak, others as a waste of bytes sent over the network.
Either reason is fine for me to disable this by default.

Impl. details:
Set suggest_enabled to false by default.
---
 chrome/browser/profiles/profile.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git chrome/browser/profiles/profile.cc chrome/browser/profiles/profile.cc
index f1a0dda49f6b6..08d0a6f47f72c 100644
--- chrome/browser/profiles/profile.cc
+++ chrome/browser/profiles/profile.cc
@@ -262,7 +262,7 @@ const char Profile::kProfileKey[] = "__PROFILE__";
 void Profile::RegisterProfilePrefs(user_prefs::PrefRegistrySyncable* registry) {
   registry->RegisterBooleanPref(
       prefs::kSearchSuggestEnabled,
-      true,
+      false,
       user_prefs::PrefRegistrySyncable::SYNCABLE_PREF);
 #if defined(OS_ANDROID)
   registry->RegisterStringPref(
-- 
2.29.1

